src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  height: 60px;
+  top: -97px;
+  left: -601px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+}
```

Comment:
Avoid fixing container sizes like width and height unless there is a specific requirement for that. It's better to let content size dictate the container size to avoid overflow or accidental scroll bars. Also, the `top` and `left` properties seem to be misused here, as they are usually used in conjunction with `position: absolute` or `position: relative` to position an element.